### ISSUE TITLE
repitition of data

### ISSUE THREAD

**Issue by jimlynnjulian** [preritdas/rottentomatoes-python#17]
Hello,

   Just ran the latest version usinf the example file in the readme page.
The list of directors is the same as the list of prominent actors.
IMDB lists only one director, Kazinski.

Thank you,
Jim Julian

**Comment by preritdas** [comment]
Thanks, will take a look once I have some time.

- Prerit

**Comment by preritdas** [comment]
@jimlynnjulian When I run it on "top gun maverick" I get:

```json
{
  "name": "Top Gun: Maverick",
  "tomatometer": 96,
  "audience_score": 99,
  "weighted_score": 97,
  "genres": [
    "Action",
    "Adventure"
  ],
  "rating": "PG-13",
  "duration": "2h 11m",
  "year": "2022",
  "actors": [
    "Tom Cruise",
    "Miles Teller",
    "Jennifer Connelly",
    "Jon Hamm",
    "Glen Powell"
  ],
  "directors": [
    "Joseph Kosinski"
  ]
}
```

This looks right to me, can you provide more info on how to replicate the issue?

**Comment by jimlynnjulian** [comment]
import rottentomatoes as rt
movie = rt.Movie('top gun')
print(movie)


RESULT:
C:\Temp\_testing\_test2>python rotompy00b.py
Top Gun: Maverick, PG-13, 2h 11m.
Released in 2022.
Directed by Tom Cruise, Miles Teller, Jennifer Connelly, Jon Hamm, Glen Powell.
Tomatometer: 96
Weighted score: 97
Audience Score: 99
Genres - ['Action', 'Adventure']
Prominent actors: Tom Cruise, Miles Teller, Jennifer Connelly, Jon Hamm, Glen Powell.

**Comment by preritdas** [comment]
Got it. Seems the issue is not with fetching directors, but in displaying a string representation of the `Movie` object. Patching right now, should be pretty quick. @jimlynnjulian

**Comment by preritdas** [comment]
Alright @jimlynnjulian, it's all fixed and published at v0.6.1.

**Comment by jimlynnjulian** [comment]
Thanks. Good work.
